### DISCUSSION TITLE
关于nginx配置的问题，很重要

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [AlistGo/alist#3267] [Q&A]
网站搭建好了，用ip访问的话就没有任何问题，包括上传，挂载在服务器本地的视频内容，都是能观看和播放，但是通过nginx反向代理后，用网址访问网站，服务器本地的视频就加载不出来了，下载显示服务器的ip禁止访问，代理过后上传文件太大会失败，但是这个我解决了，上面这个通过nginx代理后挂载在本地的视频内容没办法预览成功，下载也不行，求nginx大佬聊聊配置应该怎么改
![image](https://user-images.githubusercontent.com/78125220/216818335-81bdb35c-b8b7-4ec5-9bd8-1b40214e5d20.png)
![image](https://user-images.githubusercontent.com/78125220/216818379-c1c79985-74bc-41e6-b48c-40e83436a5e2.png)

**Comment by xrmhxj** [comment]
文档中有对nginx的配置说明https://alist.nn.ci/zh/guide/install/reverse-proxy.html

**Comment by recall704** [comment]
我也遇到了，我把 site_url 改了之后就好了。
